dom.bs

```diff
@@ -6589,6 +6591,14 @@ steps:
  <a for=Attr>value</a>.
 </ol>
 
+<p>To <dfn>verify attribute value</dfn> {{TrustedType}} or string <var>value</var> for an
+<a>attribute</a> <var>attribute</var>, given an <a for=/>Element</a> <var>element</var>:
+
+<ol>
+ <li><p>Return the result of calling <a abstract-op>get Trusted Types-compliant attribute value</a>
+ for <var>attribute</var>, with <var>element</var>, <var>value</var>. [[!TRUSTED-TYPES]]
```

Comment:
```suggestion
 for <var>attribute</var>, with <var>element</var> and <var>value</var>. [[!TRUSTED-TYPES]]
```
Although maybe we can use the more standard "given attribute, element, and value"?